base_miner/UCF/pretrained_ucf.py

```diff
@@ -0,0 +1,283 @@
+import bittensor as bt
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'  # Ignore INFO and WARN messages
+
+import random
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+from pathlib import Path
+
+import cv2
+import dlib
+import numpy as np
+import torch
+import torch.backends.cudnn as cudnn
+import torchvision.transforms as transforms
+import yaml
+from PIL import Image
+from huggingface_hub import hf_hub_download
+from imutils import face_utils
+from skimage import transform as trans
+import gc
+
+from base_miner.UCF.detectors import DETECTOR
+
+class UCF:
+    def __init__(self, config_path="./config/ucf.yaml", weights_dir="./weights/", weights_hf_repo_name="bitmind/ucf",
+                 ucf_checkpoint_name="ucf_best.pth", backbone_checkpoint_name="xception_best.pth",
+                 predictor_path = "../../bitmind/dataset_processing/dlib_tools/shape_predictor_81_face_landmarks.dat"):
```

Comment:
[Suggestion] It would be good to have paths like these in a config file (probably somewhere in the UCF dir)

Reply:
Addressed with new `base_miner/UCF/config/constants.py` file used in `pretrained_ucf.py` UCF class and across miner neurons!